degitx.go

```diff
@@ -9,6 +9,8 @@ import (
 	"log"
 	"net"
 
+	"cqfn.org/degitx/logging"
+
```

Comment:
@stepanvalyavskiy can we remove blank line here?

Reply:
sure, I'm surprised that `gofumpt` didn't do that or report.